### PR TITLE
Widget returns strings instead of printing directly

### PR DESCRIPTION
**Pull request by astromariarosa**
also, fixed access_data

### CODE REVIEW

File: get_asteroidcount.py [hunk 1]
```diff
@@ -52,22 +52,21 @@ def get_data(self,
...
-            print("Error: Cannot retrieve from the IAU Minor Planet Center.")
-            print("Check your Internet connection and minorplanetcenter.net. If both are functional, please raise an issue on Github.")
+        if self.access_data = False:
+            return "Error: Cannot retrieve from the IAU Minor Planet Center.\nCheck your Internet connection and minorplanetcenter.net. If both are functional, please raise an issue on Github."
 
         else:
-            print("We have discovered", asteroid_count['total'], "asteroids.")
+            return "We have discovered", asteroid_count['total'], "asteroids."
 
             if large:
```
Comment (timothygebhard) [inline get_asteroidcount.py]: This code is actually unreachable, because the function execution ends after the first `return` statement.

Comment (timothygebhard) [reply get_asteroidcount.py]: Also, `asteroid_count` is not available inside the `get_string` function. This is why we store it in the `self.data` variable, so it becomes accessible from everywhere within the class 🙂

Comment (timothygebhard) [reply get_asteroidcount.py]: Here's my suggestion for how the issues I've pointed out could be fixed:
```python
def get_string(self):

    string = ''

    if self.data_access:
        string = 'Error: Cannot retrieve from the IAU Minor Planet Center.\n'\
            'Check your Internet connection and minorplanetcenter.net. '\
            'If both are functional, please create an issue on Github.'
    else:
        string += f'We have discovered {self.data["total"]} asteroids.'

        if 'large' in self.data.keys():
            string += '\nOf those, there are {self.data["large"]} '\
                'near-Earth asteroids larger than 1 km wide.'
        if 'hazard' in self.data.keys():
            string += '\nCurrently, there are {self.data["hazard"]} '\
                'near-Earth asteroids that are classified as potentially '\
                'hazardous.'
    
    return string
```
Just let me know if anything is unclear or you have questions! 🙂

Comment (astromariarosa) [reply get_asteroidcount.py]: Thank you for the suggestions, I've pushed the new version. :)

**Review by ojhall94 — APPROVED** [review]

**Pull request merged**